default/stringtables/en.txt

```diff
@@ -1974,6 +1974,9 @@ Enable autosave when resigning or closing the game. In multiplayer hostless mode
 OPTIONS_DB_AUTOSAVE_HOSTLESS
 Enable autosave in hostless mode.
 
+OPTIONS_DB_AUTOSAVE_HOSTLESS_EACH_PLAYER
+Enable autosave in hostless mode after the player issued orders. Prevents losing player orders on server crash.
```

Comment:
Maybe `Enable autosave in hostless mode every time a player submits issued orders. This will prevent the loss of issued orders in the event of a server crash.`

Reply:
Fixed